invenio_vocabularies/contrib/affiliations/config.py

```diff
@@ -42,12 +42,9 @@ class AffiliationsSearchOptions(SearchOptions):
             # Aliases can sometimes be shorter, so we boost them a bit.
             "aliases^5",
             localized_title,
-            "id^2",
             # Allow to search identifiers directly (e.g. ROR)
             "identifiers.identifier",
-            "country",
```

Comment:
This might make it difficult to disambiguate funders with the same name but in different countries.

Reply:
So, the country is the abbreviation and the country_name is the full name of the country, which is what is shown in the UI too. I've also added a test for search with country.